security/advancedtls/advancedtls.go

```diff
@@ -67,6 +67,9 @@ type VerificationFuncParams = HandshakeVerificationInfo
 // future to include more information.
 type PostHandshakeVerificationResults struct{}
 
+// VerificationResults contains the information about results of
+// CustomVerificationFunc.
```

Comment:
is this moved to new line by auto formatter?

Reply:
I think so, it was the same earlier: https://github.com/grpc/grpc-go/blob/5fe2e74bf4f092addfb0a36f53964474914357a3/security/advancedtls/advancedtls.go#L57-L59

Reply:
So what's the fix here, how come tests are passed now?